esmvalcore/_recipe_checks.py

```diff
@@ -220,16 +219,88 @@ def extract_shape(settings):
                 "{}".format(', '.join(f"'{k}'".lower() for k in valid[key])))
 
 
-def valid_multimodel_statistic(statistic):
-    """Check that `statistic` is a valid argument for multimodel stats."""
-    valid_names = ['std'] + list(STATISTIC_MAPPING.keys())
+def _verify_statistics(statistics, step):
+    """Raise error if multi-model statistics cannot be verified."""
+    valid_names = ["mean", "median", "std", "min", "max"]
     valid_patterns = [r"^(p\d{1,2})(\.\d*)?$"]
-    if not (statistic in valid_names
-            or re.match(r'|'.join(valid_patterns), statistic)):
+
+    for statistic in statistics:
+        if not (statistic in valid_names
+                or re.match(r'|'.join(valid_patterns), statistic)):
```

Comment:
I don't understand why the regexp is needed once we have a set in stone list of valid statistics?

Reply:
For the percentiles I guess?

Reply:
aah right, good call!